### PR TITLE
Add emojione@3.0.1 & update emojione auto-update config

### PR DESCRIPTION
**Pull request by hare1039**
Pull request for issue: #11104
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/Ranks/emojione
 * Official website (optional, not the repository): http://www.emojione.com
 * NPM package url (optional): https://www.npmjs.com/package/emojione
 * License and its reference: MIT, https://github.com/Ranks/emojione/blob/master/LICENSE.md

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/emojione/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "emojione",
   "title": "emojione",
   "filename": "lib/js/emojione.min.js",
-  "version": "2.2.7",
+  "version": "3.0.1",
   "description": "Emoji One is a complete set of emojis designed for the web. It includes libraries to easily convert unicode characters to shortnames (:smile:) and shortnames to our custom emoji images. PNG and SVG formats provided for the emoji images.",
   "homepage": "http://www.emojione.com",
```
Comment (sufuf3) [inline ajax/libs/emojione/package.json]: please use another commit to update the homepage "https://www.emojione.com/"
let's use https

**Comment by hare1039** [comment]
@sufuf3 @pvnr0082t I update the homepage field and removed `extras/alpha-codes/`. Please review this PR again. Thanks.

**Comment by pvnr0082t** [comment]
@hare1039 Why did you ignore the review from @extend1994? That's important also.

**Comment by sufuf3** [comment]
@hare1039 Would you fix the PR as @extend1994 said https://github.com/cdnjs/cdnjs/pull/11149#pullrequestreview-34873870?
Thank you.

**Comment by hare1039** [comment]
Welcome anyone to continue on this PR.
Thank you @sashberd I'll close this PR.

**Pull request closed without merging**